RELEASING.md

```diff
@@ -117,7 +117,8 @@ Tagging the Release
    $ git checkout v$MAJOR.$MINOR.x
    $ git pull upstream v$MAJOR.$MINOR.x
    $ git checkout -b release
-   # Bump documented versions. Don't forget protobuf version
+   # Bump documented gRPC versions.
+   # Also update protoc version to match protocVersion in build.gradle.
```

Comment:
I see, we are still using https://github.com/grpc/grpc-java/blob/25022f6846ec1fdd8003789cba32d4ca981c3171/build.gradle#L60

And also need to update in examples?: https://github.com/grpc/grpc-java/search?q=protobufVersion

And  s/protocVersion/protobufVersion?  where is the `protocVersion` come from? I can not understand it.

Reply:
There are two different versions of protoc we have in our repo at a time:
 - The version we are actually using. This should be the same for the entire repo and applies to examples as well
 - The version referenced in the README, which is the version the _last release_ used

It is not appropriate to update the protobuf version being used as part of the release process. We want to continue using what we have been testing for a while. We do need to update any _documentation_ though, and that's what is discussed here.

I said protocVersion because that's the variable that matters. Yes, the code happens to have `protocVersion = protobufVersion`, but that isn't always the case.

Reply:
Yea. 
I did feel unclear when I was going through the list.
Just talking about document, we shouldn't use the new version, both the master and 1.40 should be using `3.17.2` in its document.